tilecloud_chain/configuration.py

```diff
@@ -39,7 +39,30 @@ class Argument(TypedDict, total=False):
 # Cache
 #
 # The tiles cache definition, see https://github.com/camptocamp/tilecloud-chain/blob/master/tilecloud_chain/USAGE.rst#configure-caches
-Cache = Union["CacheFilesystem", "CacheS3", "CacheMbtiles", "CacheBsddb"]
+Cache = Union["CacheFilesystem", "CacheS3", "CacheAzure", "CacheMbtiles", "CacheBsddb"]
+
+
+# Cache Azure
+#
+# Azure Blob Storage
+#
+# WARNING: The required are not correctly taken in account,
+# See: https://github.com/camptocamp/jsonschema-gentypes/issues/6
+#
+# WARNING: the Normally the types should be mised each other instead of Union.
```

Comment:
```suggestion
# WARNING: Normally the types should be mised each other instead of Union.
```
What does ``mised ``mean ?

Reply:
=> `# WARNING: Normally the types should be a mix of each other instead of Union.`
See also https://github.com/camptocamp/jsonschema-gentypes/issues/7

Reply:
=> https://github.com/camptocamp/jsonschema-gentypes/pull/113